src/common/dao/project_blob.go

```diff
@@ -34,6 +34,7 @@ func AddBlobToProject(blobID, projectID int64) (int64, error) {
 }
 
 // AddBlobsToProject ...
+// Note: pq has limitation on support parameters, the maximum length of blobs is 65535
```

Comment:
change `GetOrmer().InsertMulti(len(projectBlobs), projectBlobs)` to `GetOrmer().InsertMulti(1024, projectBlobs)`  (1024 can be other values) can make `AddBlobsToProject` to handle blobs more than 65535. 

`InsertMulti` in beego orm will split the values by bulk and insert each bulk values one by one.